ReactiveCocoaTests/Swift/SignalProducerSpec.swift

```diff
@@ -72,13 +152,42 @@ class SignalProducerSpec: QuickSpec {
 		}
 
 		describe("SignalProducer.try") {
-			pending("should run the operation once per start()") {
+			it("should run the operation once per start()") {
+				var operationRunTimes = 0
+				let operation: () -> Result<String, NSError> = {
+					operationRunTimes++
+
+					return Result<String, NSError>.Success(Box("OperationValue"))
+				}
+
+				SignalProducer.try(operation).start()
+				SignalProducer.try(operation).start()
+
+				expect(operationRunTimes).to(equal(2))
 			}
 
-			pending("should send the value then complete") {
+			it("should send the value then complete") {
+				let operationReturnValue = "OperationValue"
+				let operation: () -> Result<String, NSError> = {
+					return Result<String, NSError>.Success(Box(operationReturnValue))
+				}
+
+				let signalProducer = SignalProducer.try(operation)
+
+				startSignalProducer(signalProducer, expectSentValue: operationReturnValue, sentError: nil, complete: true)
 			}
 
-			pending("should send the error") {
+			it("should send the error") {
+				let operationError = NSError(domain: "com.reactivecocoa.errordomain", code: 4815, userInfo: nil)
+				let operation: () -> Result<String, NSError> = {
+					return Result<String, NSError>.Failure(Box(operationError))
+				}
+
+				let signalProducer = SignalProducer.try(operation)
+
+				// This is not the current behavior, but should it?
```

Comment:
In addition to @kastiglione's points, I'll note that _separate_ completed/error events is “higher fidelity.”

In other words, it's always possible to treat an error event as completion, or vice-versa—but if completion were _always_ sent, it'd be much harder to detect a successful termination (because you could only do so through the absence of errors—which you would have to track statefully).